Resources/Prototypes/Entities/Clothing/OuterClothing/suits.yml

```diff
@@ -149,6 +151,9 @@
         Slash: 0.8
         Piercing: 0.8
         Heat: 0.8
+  - type: ClothingSpeedModifier
```

Comment:
You want to change this component, not the parent. Try this instead and reverse the other stuff you did:
  - type: ClothingSpeedModifier
    sprintModifier: 1.2

Reply:
Sure

Reply:
(also remove the walk speed modifier)

Reply:
Guh